plugins/languages/sniprun.nix

```diff
@@ -6,62 +6,149 @@
   ...
 }:
 with lib;
-let
-  cfg = config.plugins.sniprun;
+helpers.neovim-plugin.mkNeovimPlugin config {
+  name = "sniprun";
+  defaultPackage = pkgs.vimPlugins.sniprun;
+  url = "https://github.com/michaelb/sniprun";
+
+  maintainers = with maintainers; [
+    traxys
+    MattSturgeon
+  ];
+
+  # TODO: Added 2024-06-17; remove 2024-09-17
+  optionsRenamedToSettings = [
+    "selectedInterpreters"
+    "replEnable"
+    "replDisable"
+    "interpreterOptions"
+    "display"
+    "liveDisplay"
+    [
+      "displayOptions"
+      "terminalWidth"
+    ]
+    [
+      "displayOptions"
+      "notificationTimeout"
+    ]
+    "showNoOutput"
+    "snipruncolors"
+    "liveModeToggle"
+    "borders"
+  ];
+
+  # https://michaelb.github.io/sniprun/sources/README.html#configuration
+  settingsOptions = {
+    selected_interpreters = helpers.defaultNullOpts.mkListOf types.str [ ] ''
+      Use those instead of the default for the current filetype.
+    '';
 
-  mkList = helpers.defaultNullOpts.mkNullable (types.listOf types.str);
-in
-{
-  options.plugins.sniprun = helpers.neovim-plugin.extraOptionsOptions // {
```

Comment:
You need to also set `deprecateExtraOptions = true` (right above the `optionsRenamedTosettings`).

Reply:
Done